### PR TITLE
Change MaterializedRow calls to have explicit long

### PR DESCRIPTION
**Pull request by raghavsethi**
In preparation for INT/SMALLINT/TINYINT, it is required that the
MaterializedRow constructors do not assume all Numbers are to be
treated as longs. This changes all callsites of the row() method to
have explicit longs.

The place that depends on this behavior is processValue in MaterializedRow

### CODE REVIEW

**Comment by dain** [comment]
Until the other types land, maybe add a check in Materialized row that the value not be an Integer, Short or Byte, so new call sites can't be created.

**Comment by electrum** [comment]
I thought we decided not to add these additional types to the engine?

File: presto-main/src/main/java/com/facebook/presto/testing/MaterializedRow.java [hunk 1]
```diff
@@ -52,6 +52,7 @@ public MaterializedRow(int precision, List<Object> values)
 
     private static Object processValue(int precision, Object value)
     {
+        checkArgument(!(value instanceof Integer || value instanceof Short || value instanceof Byte), "INT, SMALLINT and TINYINT are not yet supported");
```
Comment (haozhun) [inline presto-main/src/main/java/com/facebook/presto/testing/MaterializedRow.java]: Do you still need this?
```
         if (value instanceof Number) {
             return ((Number) value).longValue();
         }
```

**Comment by raghavsethi** [comment]
This is now part of #4711

**Pull request closed without merging**